graphql/typeDefs.ts

```diff
@@ -53,6 +54,13 @@ export default gql`
       submissionId: Int!
       content: String!
     ): Comment
+    addModule(
+      authorId: Int!
```

Comment:
when you add module, `authorId` should not be in the input. This comes from the cookie. If you do it this way, it opens up a potential security flaw where I would be able to add a module and put someone else's id to pretend like someone else created the module.